metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-windowsfeature",
-  "version": "3.0.0-rc0",
+  "version": "2.1.1",
```

Comment:
Sorry if I said it wrong or in a confusing way; if the new release will be 3.0.0, set the version to 3.0.0, then let the Travis release task bump to 3.0.1-rc0 directly in master.

Reply:
Sorry, my mistake, thought it was already on 2.1.1.

Reply:
https://github.com/voxpupuli/puppet-windowsfeature/blob/v2.1.0/metadata.json#L32 << Since that was the last Puppet 3 release, we do need to go to `3.0.0`.

Reply:
Done.